src/main/java/uk/gov/hmcts/reform/blobrouter/services/BlobVerifier.java

```diff
@@ -44,31 +45,33 @@ public VerificationResult verifyZip(String blobName, byte[] rawBlob) {
             return ok();
         } catch (DocSignatureFailureException ex) {
             logger.info("Invalid signature. Blob name: {}", blobName, ex);
-            return error("Invalid signature");
+            return error(ErrorCode.ERR_SIG_VERIFY_FAILED, "Invalid signature");
         } catch (InvalidZipArchiveException ex) {
             logger.info("Invalid zip archive. Blob name: {}", blobName, ex);
-            return error("Invalid zip archive");
+            return error(ErrorCode.ERR_ZIP_PROCESSING_FAILED, "Invalid zip archive");
         } catch (IOException ex) {
             logger.info("Error occurred when verifying file. Blob name: {}", blobName, ex);
```

Comment:
should all logs in the catch blocks be error rather than info ?

Reply:
we are logging the result in BlobProcessor where this method is called